InvoiceMaker/InvoiceMaker/FormModels/CreateWorkDone.cs

```diff
@@ -0,0 +1,50 @@
+﻿using InvoiceMaker.Models;
+using InvoiceMaker.Models.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using InvoiceMaker.Data;
+
+namespace InvoiceMaker.FormModels
+{
+    public class CreateWorkDone
+    {
+
+        private Context _context = new Context();
```

Comment:
Again, a form model class shouldn't instantiate an instance of the database context.